Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -209,6 +209,7 @@ void VertexManager::Flush()
 				if (bpmem.tevind[i].IsActive() && bpmem.tevind[i].bt < bpmem.genMode.numindstages)
 					usedtextures[bpmem.tevindref.getTexMap(bpmem.tevind[i].bt)] = true;
 
+		TextureCache::UnbindTextures();
```

Comment:
(A potential optimization would be to skip recomputing a binding if it's "the same" as it was the last time we flushed, but we don't do that with or without this PR.)